codegen.yml

```diff
@@ -23,12 +23,14 @@ generates:
   ./src/api/schema.generated.ts:
     plugins:
       - typescript
+      - named-operations-object
```

Comment:
Could we put this in operations.generated.ts? It should change based on client side usage instead of API changes.

Reply:
Good idea, I'll update right now